source/features/comment-shortcuts.tsx

```diff
@@ -0,0 +1,100 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+const shortcutKeys = new Set(['e', 'd', 'h', 'j', 'k', '+', '-', 'n']);
+const shortcutClass: Record<string, string> = {
+	e: '.rgh-edit-comment, [aria-label="Edit comment"]',
+	d: '[aria-label="Delete comment"]',
+	h: '[aria-label="Hide comment"]',
+	'+': '[data-reaction-label="+1"]',
+	'-': '[data-reaction-label="-1"]'
```

Comment:
I'd drop these honestly

Reply:
I thought it was cool, but I dropped it.